hub/config/src/main/java/uk/gov/ida/hub/config/Urls.java

```diff
@@ -36,7 +36,8 @@
         String ENABLED_IDENTITY_PROVIDERS_PATH = "/enabled";
         String ENABLED_IDENTITY_PROVIDERS_RESOURCE = IDENTITY_PROVIDER_ROOT + ENABLED_IDENTITY_PROVIDERS_PATH;
         String ENABLED_IDENTITY_PROVIDERS_PARAM_PATH = ENTITY_ID_PATH_PARAM + "/enabled"; //deprecated
-        String ENABLED_ID_PROVIDERS_FOR_LOA_PATH = ENTITY_ID_PATH_PARAM + LEVEL_OF_ASSURANCE_PATH_PARAM + "/enabled";
+        String ENABLED_IDENTITY_PROVIDERS_PARAM_PATH_RESOURCE = IDENTITY_PROVIDER_ROOT + ENABLED_IDENTITY_PROVIDERS_PARAM_PATH; // deprecated
```

Comment:
I think you can `@Deprecated` a field if that sounds fun?

Reply:
You can! How fun. Added to the review commit